exercises/sampler-built-in-types/test.ml

```diff
@@ -0,0 +1,51 @@
+open Test_lib
+open Report
+
+(*A first way to grade this function with randomly generated tests 
+  is by using the default sampler.*)
+let exercise_1 =
+  Section ([ Text "Function: "; Code "identity" ],
+           test_function_1_against_solution
+             [%ty: int -> int] "identity"
+             ~sampler:(fun () -> Random.int 32)
+             ~gen:5 [0]
+          )
+
+
+(*Using the optional argument ~sampler*)
+let exercise_2 =
+  Section ([ Text "Function: "; Code "pi1" ],
+           test_function_2_against_solution
+             [%ty: int -> int -> int] "pi1"
+             ~sampler:(fun () -> Random.int 31 + 12, Random.int 31 + 12)
+             ~gen:5
+             []
+          )
+
+
+(*Redefine sampler*)
```

Comment:
Elaborate on this. Recall that the mapping between types and samplers is based on a naming convention.